build.gradle

```diff
@@ -718,10 +718,7 @@ class JDKDetails {
     private String arch
 
     JDKDetails(versionYml, osName, jdkArch) {
-        revision = versionYml.bundled_jdk.revision
-        build = versionYml.bundled_jdk.build
-        vendor = versionYml.bundled_jdk.vendor
```

Comment:
I am wondering can we keep the old path in case we need to pin jdk version for quick fix/ release. I believe most of the time using the version from the endpoint is fine. When `revision`, `vendor` and `build` is defined, they take precedence.